README.md

```diff
@@ -95,6 +95,7 @@ _See also: [Restic's list of Linux backup software](https://github.com/restic/ot
 - [Dar](http://dar.linux.free.fr/) - Which stands for Disk ARchive, is a robust and rich featured archiving and backup software of the tar style. ([Source Code](https://github.com/Edrusb/DAR)) `GPL-2.0` `C++`
 - [Duplicati](https://www.duplicati.com) - Backup client that securely stores encrypted, incremental, compressed backups on cloud storage services and remote file servers. ([Source Code](https://github.com/duplicati/duplicati)) `LGPL-2.1` `C#`
 - [Duplicity](https://duplicity.gitlab.io/) - Encrypted bandwidth-efficient backup using the rsync algorithm. ([Source Code](https://gitlab.com/duplicity/duplicity)) `GPL-2.0` `Python`
+- [Open Archiver](https://docs.openarchiver.com) - Email archiving solution with full-text search and eDiscovery search features. ([Source Code](https://github.com/LogicLabs-OU/OpenArchiver)) `AGPL-3.0` `TypeScript`
```

Comment:
```suggestion
- [Open Archiver](https://docs.openarchiver.com) - Email archiving solution with full-text search and eDiscovery search features. ([Source Code](https://github.com/LogicLabs-OU/OpenArchiver)) `AGPL-3.0` `Docker`
```

Is a requirement to run the program https://docs.openarchiver.com/user-guides/installation.html#prerequisites
The actual programming language does not matter that much to users